README.md

```diff
@@ -1,55 +1,149 @@
-# COVID-19 CLI
+# CoVID-19 CLI
 
 <img src="https://badges.frapsoft.com/os/v1/open-source.png?v=103"> <a href="LICENSE"><img alt="GitHub" src="https://img.shields.io/github/license/dmahajan980/covid19"></a>
 
 A command-line interface to check the current statistics of COVID-19 of the world or a particular country.
 
-## TABLE OF CONTENTS
-1. [Getting Started](#getting-started)
-2. [Usage](#usage)
-3. [Contributing](#contributing)
-5. [License](#license)
+## Table of Contents
 
-## GETTING STARTED
+1.  [Getting Started](#getting-started)
+2.  [Usage](#usage)
+3.  [Contributing](#contributing")
+4.  [License](#license)
 
-1. Clone or download the repository.
+## Getting Started
 
-2. Install all the dependencies using the following command
+To get started with the CLI is very simple
 
-   ```bash
-   npm install
-   ```
-   
-3. Create a symlink of the repository in the global folder
+1.  You can choose to clone or download this repository.  
+    To clone this repository, the command is  
 
-   ```bash
-   sudo npm link
-   ```
+    ```bash
+    git clone https://github.com/dmahajan980/covid19.git
+    ```
 
-## USAGE
+2.   Now you need to install the dependencies. To do so, run the command
+      ```bash
+      npm install
+      ```
 
-The CLI will return the latest global statistics about COVID-19 in a tabular format.
+3.   The last step is to add a symbolic link to the global folder.  
+    If you are on Windows, start `cmd.exe` as Administrator and run the command
+      ```bash
+      npm link
+      ```
+      If you are on macOS/Linux, the command is
+      ```bash
+      sudo npm link
+      ```
 
-   ```bash
-   covid19 stats
-   ```
+And we are ready to go!
 
-The CLI can also return the COVID-19 statistics of a country, if specified, using one of the following commands.
+## Usage
 
-   ```bash
-   covid19 stats --country <countryName>
-   ```
+This CLI is very easy to use.
 
-   ```bash
-   covid19 stats -c <countryName>
-   ```
+*   **covid19**  
+    This is the name of the package that is used to invoke the CLI.  
+    Flags are available for this command.  
 
-The supported formats for `countryName` are ISO2, ISO3 or a single-word name.
+    <table style="height: 5px; width: 600px;" border="black">
 
-## CONTRIBUTING
+    <tbody>
+
+    <tr style="border-color: black;">
+
+    <td style="width: 150px; text-align: center; border-color: black;"><span style="text-decoration: underline;">flag</span></td>
```

Comment:
You can use `th` instead of `td` for all the column headings so that the word is displayed as a heading. Don't forget to "capitalize" the headings as well.